### ISSUE TITLE
Can not ping camera source, no source found! Video is not showing up

### ISSUE THREAD

**Issue by geritt81** [seydx/homebridge-camera-ui#449]
### Current Situation

When using Camera-UI plugin the video stream is not showing in home app, it says camera offline when opening video. But snapshot picture is showing in the app and on the Homebridge plugin page i can view the video without issues. 
But in home app and portal video is not working at all.

Tried also FFmpeg-camera plugin, video works without issues there. with basically same configuration. Used two different IP cameras, played with the settings, plugin versions, always same problem, always "can not ping camera source, no source found"

### Logs

```markdown
[10/02/2022, 08:08:17] [CameraUI] Frontdoor1: Can not ping camera source, no source found (-i rtsp://192.168.1.7:554/onvif1)
[10/02/2022, 08:08:17] [CameraUI] Frontdoor1: Starting video stream: 640x360, 20 fps, 614 kbps 
[Frontdoor1 11AF] The image snapshot handler for the given accessory is slow to respond! See https://git.io/JtMGR for more info.
```

### Configuration

```json
{
            "name": "CameraUI",
            "port": 8081,
            "atHomeSwitch": false,
            "theme": "auto",
            "language": "Auto",
            "debug": false,
            "mqtt": {
                "active": false,
                "tls": false,
                "port": 1883
            },
            "http": {
                "active": false,
                "port": 7272,
                "localhttp": false
            },
            "ssl": {
                "active": false
            },
            "options": {
                "videoProcessor": "/usr/local/lib/node_modules/homebridge-camera-ui/node_modules/ffmpeg-for-homebridge/ffmpeg"
            },
            "cameras": [
                {
                    "name": "Frontdoor1",
                    "motion": false,
                    "doorbell": false,
                    "motionTimeout": 15,
                    "unbridge": true,
                    "videoConfig": {
                        "source": "-i rtsp://192.168.1.7:554/onvif1",
                        "maxStreams": 4,
                        "maxWidth": 640,
                        "maxHeight": 360,
                        "maxFPS": 20,
                        "maxBitrate": 614,
                        "forceMax": true,
                        "vcodec": "libx264",
                        "stimeout": 0,
                        "encoderOptions": "-preset ultrafast -tune zerolatency",
                        "audio": false,
                        "debug": true,
                        "stillImageSource": "-i rtsp://192.168.1.7:554/onvif1",
                        "subSource": "-i rtsp://192.168.1.7:554/onvif1"
                    },
                    "prebufferLength": 4,
                    "videoanalysis": {
                        "active": false
                    },
                    "smtp": {
                        "email": "Frontdoor1"
                    },
                    "mqtt": {},
                    "hsv": false
                },
                {
                    "name": "Carport1",
                    "motion": false,
                    "motionTimeout": 15,
                    "unbridge": true,
                    "videoConfig": {
                        "source": "-i rtsp://192.168.1.12:554/11",
                        "stillImageSource": "-i rtsp://192.168.1.12:554/11",
                        "maxStreams": 2,
                        "vcodec": "copy",
                        "stimeout": 0,
                        "audio": false,
                        "subSource": "-i rtsp://192.168.1.12:554/11"
                    },
                    "prebufferLength": 4,
                    "videoanalysis": {
                        "active": false
                    },
                    "smtp": {
                        "email": "Carport1"
                    },
                    "mqtt": {},
                    "hsv": false
                }
            ],
            "platform": "CameraUI",
            "smtp": {
                "active": false,
                "port": 2727,
                "space_replace": "+"
            },
            "ftp": {
                "active": false,
                "useFile": false,
                "port": 5050
            }
        }
```

### Environment

- camera-ui: v5.0.16
- Homebridge: v1.4.0
- Node: v16.13.2
- npm: v8.1.2

**Comment by WillyWurst** [comment]
Same issue here. But I am pretty much sure that with the current version (5.0.16) with this plugin it was running before. Maybe an issue with iOS 15.4 update????

**Comment by seydx** [comment]
should be fixed in v5.0.17-beta.1
